### DISCUSSION TITLE
visual testing html report

### DISCUSSION THREAD

**Discussion by radhakrishnanakireddy** [seleniumbase/SeleniumBase#1096] [Ideas]
Hi,
Firstly Thanks a lot for providing support for visual testing and we would like to try it
I see there is chance to improve the visual testing report,
I see screenshot link in report where we can display combined baseline image and actual image side by side👍 

could you please check the chance to have improvement for it if possible

**Comment by mdmintz** [comment]
@radhakrishnanakireddy Thank you so much for your support.
The improvements to visual testing have been completed in version ``2.2.4``:
https://github.com/seleniumbase/SeleniumBase/releases/tag/v2.2.4

**Reply by radhakrishnanakireddy** [reply]
Thanks a lot for considering it!
